.github/pull_request_template.md

```diff
@@ -9,3 +9,6 @@ If some of the following don't apply, delete the relevant line.
 - [ ] Added support on fleet's osquery simulator `cmd/osquery-perf` for new osquery data ingestion features.
 - [ ] Added/updated tests
 - [ ] Manual QA for all new/changed functionality
+  - For Orbit changes:
+    - [ ] Manual QA must be performed in the three main OSs, macOS, Windows and Linux.
+    - [ ] Auto-update manual QA, from released version of Orbit to new version (see [tools/tuf/test](../tools/tuf/test/README.md)).
```

Comment:
this is great, should we add a check for compatibility? a.k.a. "use the capabilities API"

Reply:
and probably change 
```diff
-  - For Orbit changes:
+ - For Orbit and Fleet Desktop changes:
```

Reply:
> this is great, should we add a check for compatibility? a.k.a. "use the capabilities API"

I believe this kind of advice should actually be on a development guide, not here.
If the developer didn't use that properly then the proposed "Auto-update manual QA test" will fail.